lib/change_stream_simulator.js

```diff
@@ -0,0 +1,244 @@
+var Timestamp = require('mongodb-core').BSON.Timestamp;
+var MongoNetworkError = require('mongodb-core').MongoNetworkError;
+var supportedStages =  ['$match', '$project', '$addFields', '$replaceRoot', '$redact'];
```

Comment:
nit: totally just convention, but generally constants like this are capitalized, e.g. `SUPPORTED_STAGES`